DESCRIPTION

```diff
@@ -17,6 +17,8 @@ Imports:
     dplyr (>= 0.2),
     stringi,
     lazyeval
+Suggets:
```

Comment:
Ah I just discovered that too and fixed it (in a slightly different way). Could you please merge/rebase?